src/foundry/foundry.js/applications/sidebarTabs/combatTracker.d.ts

```diff
@@ -2,184 +2,188 @@
 
-  /**
-   * Display a dialog which prompts the user to enter a new initiative value for a Combatant
-   */
-  protected _onConfigureCombatant(li: JQuery): void;
-}
+    /**
+     * Handle mouse-down event on a combatant name in the tracker
+     * @param event - The originating mousedown event
+     * @returns A Promise that resolves once the pan is complete
+     */
+    protected _onCombatantMouseDown(event: JQuery.ClickEvent): Promise<void> | void;
 
-declare namespace CombatTracker {
-  type Data = {
-    user: User;
-    started: boolean;
-    settings: Combat.ConfigValue;
-  } & (
-    | {
-        combats: [];
-        currentIndex: -1;
-        combatCount: 0;
-        hasCombat: false;
-        combat: null;
-        turns: [];
-        previousId: null;
-        nextId: null;
-        control: false;
-      }
-    | {
-        combats: Combat[];
-        currentIndex: number;
-        combatCount: number;
-        hasCombat: true;
-        combat: Combat;
-        turns: Turn[];
-        previousId: string | null;
-        nextId: string | null;
-        round: number;
-        turn: number;
-        control: boolean;
-      }
-  );
+    /**
+     * Handle mouse-hover events on a combatant in the tracker
+     */
+    protected _onCombatantHover(event: JQuery.MouseEnterEvent): void;
 
-  interface Options extends SidebarTab.Options {
     /**
-     * @defaultValue `'combat'`
+     * Handle mouse-unhover events for a combatant in the tracker
      */
-    id: string;
+    protected _onCombatantHoverOut(event: JQuery.MouseLeaveEvent): void;
 
     /**
-     * @defaultValue `'templates/sidebar/combat-tracker.html'`
+     * Default folder context actions
      */
-    template: string;
+    protected _contextMenu(html: JQuery): void;
 
     /**
-     * @defaultValue `'Combat Tracker'`
+     * Get the sidebar directory entry context options
+     * @returns The sidebar entry context options
      */
-    title: string;
+    protected _getEntryContextOptions(): ContextMenu.Item[];
 
     /**
-     * @defaultValue `['.directory-list']`
+     * Display a dialog which prompts the user to enter a new initiative value for a Combatant
      */
-    scrollY: string[];
+    protected _onConfigureCombatant(li: JQuery): void;
   }
 
-  type Turn = foundry.utils.Duplicated<Combat.Combatant> & {
-    effects: Set<unknown>;
-    active: boolean;
-    css: string;
-    hasRolled: boolean;
-    hasResource: boolean;
-  };
+  namespace CombatTracker {
+    type Data = {
+      user: User;
+      started: boolean;
+      settings: Combat.ConfigValue;
+    } & (
+      | {
+          combats: [];
+          currentIndex: -1;
+          combatCount: 0;
+          hasCombat: false;
+          combat: null;
+          turns: [];
+          previousId: null;
+          nextId: null;
+          control: false;
+        }
+      | {
+          combats: StoredDocument<InstanceType<ConfiguredDocumentClass<typeof Combat>>>[];
```

Comment:
for the base implementation, this is correct, I think, but I'm not sure I would prevent modules / systems to do it differently...